diff --git a/node_modules/nextra-theme-docs/dist/layout.js b/node_modules/nextra-theme-docs/dist/layout.js
index 57a511f..2347241 100644
--- a/node_modules/nextra-theme-docs/dist/layout.js
+++ b/node_modules/nextra-theme-docs/dist/layout.js
@@ -27,7 +27,7 @@ const Layout = (t0) => {
     const {
       data: t12,
       error
-    } = LayoutPropsSchema.safeParse(themeConfig);
+    } = LayoutPropsSchema.safeParse({children, ...themeConfig}); // Fixed for Uncaught Error: ✖ Invalid input: expected nonoptional, received undefined
     data = t12;
     if (error) {
       throw z.prettifyError(error);
